plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -910,6 +912,255 @@ public void testSettingChangeDataFeedEnabledProperty()
                 .contains("change_data_feed_enabled = true");
     }
 
```

Comment:
There seems to be missing a test for calling the `table_changes` function on a table which doesn't have CDF enabled.

Also it would be good to have a test for a table where CDF is not initially enabled, some changes are performed on the table and then the CDF is enabled.

Reply:
Probably because I didn't implement this check at all...

Reply:
and this is hard :|